src/sentry/models/monitorcheckin.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import, print_function
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    GUIDField,
+    Model,
+    BaseManager,
+    BoundedPositiveIntegerField,
+    EncryptedJsonField,
+    FlexibleForeignKey,
+    sane_repr,
+)
+
+
+class CheckInStatus(object):
+    UNKNOWN = 0
+    OK = 1
+    ERROR = 2
+    IN_PROGRESS = 3
+
+    FINISHED_VALUES = (OK, ERROR)
+
+    @classmethod
+    def as_choices(cls):
+        return (
+            (cls.UNKNOWN, 'unknown'),
+            (cls.OK, 'success'),
+            (cls.ERROR, 'failure'),
+            (cls.IN_PROGRESS, 'in_progress'),
+        )
+
+
+class MonitorCheckIn(Model):
+    __core__ = True
+
+    guid = GUIDField()
```

Comment:
unique?

Reply:
i had put this in the field itself but im actually going to pull it out as that didnt make sense